robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -90,10 +88,20 @@ public static class CORAL_RUNNER {
       .openLoopRampRate(.25)
       .voltageCompensation(12)
       .smartCurrentLimit(40, 40);
-    public static final Dimensionless AXIS_MAX_SPEED = Units.Percent.of(20);
+    public static final Dimensionless AXIS_MAX_SPEED = Units.Percent.of(50);
     public static final Time DEBOUNCE_TIME_SECONDS = Units.Seconds.of(.03);
     public static final Time SENSOR_PERIODIC_TIME = Units.Milliseconds.of(5);
     public static final Time SENSOR_PERIODIC_OFFSET_TIME =
       Units.Milliseconds.of(3);
   }
+
+  public static class ALGAE_KNOCKER {
+
+    public static final SparkBaseConfig MOTOR_CONFIG = new SparkMaxConfig()
+      .idleMode(IdleMode.kBrake)
+      .openLoopRampRate(.25)
+      .voltageCompensation(12)
+      .smartCurrentLimit(40, 40);
+    public static final Dimensionless DEALGAE_SPEED = Units.Percent.of(50);
```

Comment:
Let's stick in and underscore, `DE_ALGAE_SPEED` Makes the prefix which indicates what the speed is for more clear.